src/__experimental__/components/input/input.component.js

```diff
@@ -27,6 +27,7 @@ const Input = React.forwardRef(
     const context = useContext(InputContext);
     const groupContext = useContext(InputGroupContext);
     const deferredTimeout = useRef(null);
+    // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
Eslint started to show an error that conditionals are not ok on `useRef` so I've added this comment/eslint. I though it will be done in other ticket

Reply:
That's strange because the lint CI would be failing on every PR if that was the case, and the eslint config hasn't changed in this PR!

Have you got any eslint rules in your IDE which might be accidentally being used here?

Reply:
I've pulled the branch down locally and this doesn't flag, you should remove this